src/c++/perf_analyzer/test_request_rate_manager.cc

```diff
@@ -169,6 +174,28 @@ class TestRequestRateManager : public TestLoadManagerBase,
     }
   }
 
+  void TestSchedule(double rate, PerfAnalyzerParameters params)
+  {
+    PauseWorkers();
+    GenerateSchedule(rate);
+
+    std::chrono::nanoseconds max_test_duration{params.measurement_window_ms *
+                                               1000000 * params.max_trials};
+    std::chrono::nanoseconds expected_interval{int(1000000000 / rate)};
```

Comment:
rewrite magic number into variable?

Reply:
magic numbers exist in perf_utils.h. I'm using them now